tests/jupyterhub_api/test_check_session.py

```diff
@@ -0,0 +1,136 @@
+"""
```

Comment:
It's good you document how to use this, but add the docstrings after the class definition. This way it's proper Python and makes it easier to inspect (you can do `help()` on the class and get the documentation. More info: https://docs.python.org/3/library/functions.html#help)